tests/unit.cpp

```diff
@@ -406,6 +406,91 @@ TEST(db, collection_drop_values) {
     EXPECT_TRUE(db.clear());
 }
 
+TEST(db, multiple_collection) {
```

Comment:
What does this do? Is it the minimal unit case possible?

Reply:
I create more than one collection and write to them, then delete some of them. I check that the deletion of some does not delete the content of others.